shared/templates/static/ansible/no_empty_passwords.yml

```diff
@@ -3,9 +3,11 @@
 # strategy = configure
 # complexity = low
 # disruption = medium
-
 - name: "Prevent Log In to Accounts With Empty Password"
   replace:
     dest: /etc/pam.d/system-auth
     regexp: 'nullok\s*'
     replace: ''
+  tags:
```

Comment:
Actually, they don't. I only use check_mode no for tasks with no side effects that gather variables.  That's what it's intended for.